pkg/providers/ingress/ingress.go

```diff
@@ -428,7 +428,20 @@ func (c *ingressController) ResourceSync() {
 		if !c.namespaceProvider.IsWatchingNamespace(key) {
 			continue
 		}
+
 		ing := kube.MustNewIngress(obj)
+		valid := c.isIngressEffective(ing)
+		if valid {
+			log.Debugw("ingress add event arrived",
+				zap.Any("object", obj),
+			)
+		} else {
+			log.Debugw("ignore noneffective ingress add event",
+				zap.Any("object", obj),
+			)
+			return
+		}
```

Comment:
We can simplify the code here, right?

Reply:
Are you means to summarize these "valid judge" logic to common codes?

Reply:
yes, we don't need judge `valid` and print log.

Reply:
> yes, we don't need judge `valid=true` and print log.

Ok, i'll modify it